examples/contracts/Cargo.toml

```diff
@@ -9,7 +9,7 @@ publish = false
 repository = { workspace = true }
 description = "Fuel Rust SDK contract examples."
 
-[dev-dependencies]
+[dependencies]
```

Comment:
Examples didn't build under newer rust versions because you cannot (still) enable features on dev deps via feature flags of your own (and it seems we tried to do that in the past but failed silently... until now).